openpype/settings/defaults/project_settings/fusion.json

```diff
@@ -27,6 +27,18 @@
                 "farm_rendering"
             ],
             "image_format": "exr"
+        },
```

Comment:
Please also bump a `version.py` since those changes will require it even for testing.

Reply:
I merged develop, that should bump `openpype/version.py`.